helpers/controllers/authController.ts

```diff
@@ -61,6 +62,11 @@ export const login = async (_parent: void, arg: Login, ctx: Context) => {
     const cliToken = { id: user.id, cliToken: user.cliToken }
 
     session.userId = user.id
+    ctx.res.setHeader(
+      'Set-Cookie',
+      `loggedIn=true; expires=${new Date(Date.now() + ONE_WEEK)}; Path=/`
+    )
```

Comment:
Why not directly set this cookie client-side instead of making the server set it through the response? It kinda makes more sense too because it's only a client-side check. You can do it with `document.cookie`.